tests/l2/dsa/README.md

```diff
@@ -0,0 +1,34 @@
+### Verify Intel® Data Streaming Accelerator (DSA) Technology provisioning
+This workload runs [accel-config](https://github.com/intel/idxd-config) sample tests using RedHat built and distributed accel-config RPM packages from the rhel-9-for-x86_64-baseos-rpms repo. Refer to the [accel config readme](https://github.com/intel/idxd-config/blob/stable/README.md) for more details. 
+
+*	Build the workload container image
+
+Please replace the credentials in buildconfig yaml with your RedHat account login credentials. 
+
+```
+$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/tests/l2/dsa/dsa_build.yaml 
+```
+
+* Create SCC intel-dsa-scc for Intel DSA based workload, if this SCC is not created   
+  
+```
+$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/security/dsa_scc.yaml
+```
+      
+* Create the intel-dsa service account to use intel-dsa-scc
+  
+```
+$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/security/dsa_rbac.yaml
+```
+
+* Deploy the accel-config workload job with intel-dsa service account
+  
+```
+$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/tests/l2/dsa/dsa_job.yaml
+```
+
+* Check the results.
+``` 
+  $ oc get pods
+  intel-dsa-workload-244xm   0/1     Completed   0          3m12s
```

Comment:
we should also to show the job running result from the log.

Reply:
added